src/components/TaskCard/index.jsx

```diff
@@ -3,18 +3,21 @@ import PropTypes from "prop-types";
 import classNames from "classnames";
 import { useDispatch } from "react-redux";
 import { deleteTask, completeTask } from "store/actions";
+import { getDateDifference } from "utils/helpers/getDateDifference";
+import "components/TaskCard/index.scss";
```

Comment:
import styles at bottom of all imports. [reference](https://github.com/azo-wtag/todo-app-react/pull/1#discussion_r1197359972)